standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -369,32 +369,36 @@ public void setConf(Configuration conf){
     this.conf = conf;
 
     synchronized (TxnHandler.class) {
+      int maxPoolSize = MetastoreConf.getIntVar(conf, ConfVars.CONNECTION_POOLING_MAX_CONNECTIONS);
```

Comment:
minor: maxPoolSize & connectionTimeoutMs could me moved outside of synchronized block

Reply:
Done